deps/softwarecontainer-dependencies.sh

```diff
@@ -44,16 +44,27 @@ function install {
 # For softwarecontainer
 install libdbus-1-dev libglibmm-2.4-dev libglibmm-2.4 \
         unzip bridge-utils lcov libjansson-dev libjansson4 \
-        dbus-x11 libcap-dev libtool python3-dev
+        dbus-x11 libcap-dev libtool python3-dev \
+        seccomp libseccomp-dev
 
-apt-get remove --allow-downgrades --allow-remove-essential --allow-change-held-packages -fuy lxcfs lxc2 lxc-dev lxc-common
+apt-get remove --allow-downgrades --allow-remove-essential --allow-change-held-packages -fuy lxcfs lxc2 lxc-dev lxc-common \
+        liblxc-common liblxc-dev lxc1 liblxc1 lxc-templates lxc-tests lxc-utils python3-lxc
 
 # Download and install lxc
 rm -rf lxc
 git clone git://github.com/lxc/lxc -b lxc-3.1.0
 cd lxc
 
 ./autogen.sh
-./configure --prefix=/usr --enable-capabilities --enable-python
+./configure --prefix=/usr --enable-capabilities --enable-python --enable-apparmor --enable-seccomp
```

Comment:
This is not mentioned in the commit message, so please put into separate commit to highlight that these features are enabled in the config.